RoomMe.Webservice/RoomMe.Webservice/Controllers/AccountController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using RoomMe.Webservice.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace RoomMe.Webservice.Controllers
+{
+    [RoutePrefix("api/account")]
+    public class AccountController : ApiController
+    {
+
+        private RoomMeWebserviceContext db = new RoomMeWebserviceContext();
+
+        // GET: api/Account
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET: api/Account/5
+        public string Get(int id)
+        {
+            return "value";
+        }
+
+        [Route("/register")]
+        // POST: api/Account
+        public async Task<IHttpActionResult> Register([FromBody] string email, [FromBody] string name, [FromBody] int age, [FromBody] string bio, [FromBody] int gender, [FromBody] string phone, [FromBody] string authToken)
+        {
+            User newUser = new User()
+            {
+                Email = email,
+                Name = name,
+                Age = age,
+                Bio = bio,
+                Gender = (Gender) gender,
+                PhoneNumber = phone,
+                Status = Status.NeedsHousingAndRoommate,
+                AuthToken = authToken
+            };
```

Comment:
Looks good, but we can get a whole lot more data from Facebook. We just have to decide what will be useful.